examples/PublishHelloWorld/main.c

```diff
@@ -58,7 +58,7 @@ int main(int args, char** argv)
 
     // Create entities
     mrObjectId participant_id = mr_object_id(0x01, MR_PARTICIPANT_ID);
-    const char* participant_ref = "default participant";
+    const char* participant_ref = "default_xrce_participant_profile";
```

Comment:
I supposed that is necessary to change it also into:
`examples/Deployment/configurator_client.c:134`
`examples/Deployment/configurator_client.c:158`
`test/integration/interaction/ClientInteraction.hpp:42`

Reply:
Done!